docs/enhancements/api-for-skipping-validations.md

```diff
@@ -0,0 +1,148 @@
+---
+title: api-for-skipping-validations
+authors:
+  - "@pmaidmen"
+creation-date: 2023-01-08
+last-updated: 2023-01-08
+---
+
+# Allow validations to be skipped
+
+# Summary
+
+We need new api that will allow us to skip cluster/host validations.
```

Comment:
do we want to skip it for a specific cluster or for all clusters? if it's for a specific cluster - we should add the cluster id to the POST API.
if it's for all clusters - we have something similar today - https://github.com/openshift/assisted-service/blob/master/openshift/template.yaml#L93 - let's mention it in this enhancement